test/provision/k8s_install.sh

```diff
@@ -62,10 +75,16 @@ KUBEADM_SLAVE_OPTIONS=""
 KUBEADM_OPTIONS=""
 
 case $K8S_VERSION in
-    "1.7"|"1.8")
+    "1.7")
+        K8S_FULL_VERSION="1.7.11"
```

Comment:
Why is this needed? I would like to have the behaviour that we had in the past.

Reply:
Unfortunately, you cannot tell `zypper` to install just `1.7` version. You need to specify the whole version that the concrete RPM package has. That's sad, but I cannot do anything about that.

Reply:
It means, that `zypper in kubelet=1.7.11` works and `zypper in kubelet=1.7` doesn't. Same with wildcards.